config/routes.rb

```diff
@@ -40,7 +40,6 @@
   end
 
   get 'samples/:id/fasta/:tax_level/:taxid/:hit_type', to: 'samples#show_taxid_fasta'
-  get 'samples/:id/alignment/:taxon_info', to: 'samples#show_taxid_alignment'
```

Comment:
Nit: Also remove `show_taxid_alignment` from READ_ACTIONS in samples_controller.

Reply:
it's already gone, i removed it in the last diff and forgot to include this change in the diff